core/subpopulation.cpp

```diff
@@ -730,10 +730,6 @@ void Subpopulation::GenerateParentsToFit(slim_age_t p_initial_age, double p_sex_
 			parent_individuals_.push_back(individual);
 		}
 	}
-	if (recording_tree_sequence)
-	{
-		sim.AddIndividualsToTable(parent_individuals_.data(), parent_individuals_.size(), nullptr, SLIM_TSK_INDIVIDUAL_FIRST_GEN);
```

Comment:
OK, so... how do individual table entries get added now?  Do we just say that there are no individual table entries for the first gen unless the user calls treeSeqRememberIndividuals() on them?

Reply:
yes!